client/src/main/java/io/spine/client/ActorRequestFactory.java

```diff
@@ -137,11 +135,13 @@ protected CommandContext createCommandContext() {
      * <p>Sets the timestamp value to the
      * {@linkplain io.spine.base.Time#getCurrentTime() current time}.
      */
+    @SuppressWarnings("CheckReturnValue") // calling builder
     ActorContext actorContext() {
-        final ActorContext.Builder builder = ActorContext.newBuilder()
-                                                         .setActor(actor)
-                                                         .setTimestamp(getCurrentTime())
-                                                         .setZoneOffset(zoneOffset);
+        ActorContext.Builder builder = ActorContext
```

Comment:
@dmdashenkov, it's the new layout for calling builders. In Java 11, we'd  have `var` instead of type.